### PR TITLE
Better installation performances

### PR DESCRIPTION
**Pull request by Quetzacoalt91**
SPEED

| Questions | Answers |
| --- | --- |
| Branch? | develop |
| Description? | By changing a few lines and upgrading the lib Archive_Tar, we can get 31% of performance boost on some case. |
| Type? | improvement |
| Category? | Install / CO |
| BC breaks? | Nope |
| Deprecations? | Nope |
| How to test? | To have the best results, execute your installation in CLI mode, not with the install wirzard. You can see the difference if you need to clean the database during the installation. |

Blackfire diff results:
blackfire improvement

### CODE REVIEW

**Comment by firstred** [comment]
PDO mandatory? Really?

**Comment by firstred** [comment]
Go for prepared statements: http://php.net/manual/en/pdo.prepared-statements.php! :smiley: 
Proof of Concept here: https://github.com/PrestaShop/PrestaShop/pull/5302

**Comment by Shudrum** [comment]
PDO is mandatory because of Doctrine and Dbal.

We will not change everything to prepared statements because it is useful if we repeat the call to the prepared request or to automatically escape all values.

During the transitions to Symfony, we will move to Dbal, but we will not change all the requests just to change the requests.

**Comment by firstred** [comment]
> We will not change everything to prepared statements because it is useful if we repeat the call to the prepared request or to automatically escape all values.

Indeed, that makes sense to me. I've noticed that automatically preparing `INSERT` and `UPDATE` statements can even lead to a decline in performance.

**Comment by mickaelandrieu** [comment]
Thank you @Quetzacoalt91 !

File: install-dev/controllers/http/system.php [hunk 1]
```diff
@@ -94,7 +94,7 @@ public function display()
                         'upload' => $this->l('Cannot upload files'),
                         'system' => $this->l('Cannot create new files and folders'),
                         'gd' => $this->l('GD library is not installed'),
-                        'mysql_support' => $this->l('MySQL support is not activated'),
+                        'pdo_mysql' => $this->l('PDO MySQL extension is not loaded'),
```
Comment (mickaelandrieu) [inline install-dev/controllers/http/system.php]: even if PHP PDO extension is installed, the check goes wrong :/

Comment (rGaillard) [inline install-dev/controllers/http/system.php]: #5716 the issue

**Pull request merged**